pkg/kubeapiserver/options/authentication.go

```diff
@@ -622,10 +622,7 @@ func (o *BuiltInAuthenticationOptions) ApplyTo(authInfo *genericapiserver.Authen
 		authenticatorConfig.CustomDial = egressDialer
 	}
 
-	authInfo.Authenticator, openAPIConfig.SecurityDefinitions, err = authenticatorConfig.New()
-	if openAPIV3Config != nil {
```

Comment:
openapiV3Config can no longer be nil?

Reply:
technically true since it's GA and we'll be [removing the feature flag in this release](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/features/kube_features.go#L292C2-L292C96). That belongs in a separate PR though, added the `openAPIV3Config != nil` check back in. Thanks for catching this.